README.md

```diff
@@ -148,23 +148,24 @@ The functions will be loaded in the same order as they are inside the array.
 -------------------------------------------------------
 <a name="after"></a>
 
-### app.after(func([context], [done]), [cb])
+### app.after(func(error, [context], [done]), [cb])
```

Comment:
Why not, in this way the user can handle the error inside the after.
Example:
If plugin 1 fails => load plugin 2.